src/main/java/com/blackducksoftware/integration/jira/issue/ui/JiraFieldUtils.java

```diff
@@ -98,19 +104,13 @@ private static boolean isBdsCustomField(final Field field) {
          // @formatter:on
     }
 
-    private static void addEligibleSystemFields(final BlackDuckJiraLogger logger, final FieldManager fieldManager, final Fields targetFields) {
```

Comment:
Was this deleted because it was not being used?

Reply:
Correct!